FFmpegChapter id needs to be a long

**Describe the bug**
While attempting to convert a 4k video file (about 2:20:00 in length, 31.22GB) in my spring server, the API call fails because `FFProbe` attempts to deserialize an `FFmpegProbeResult` on line 114. The `FFmpegChapter` has a field 'id' which is defined as an int, however this video file happens to have a chapterID which is larger than 32 bits. I've solved the issue for my video file by changing id to be a long.

**To Reproduce**
```java
FFmpegBuilder builder = new FFmpegBuilder();
final String inputName = "Input.mkv"
final String outputName = "output/" + inputName;
builder.setInput(inputName)     // Filename, or a FFmpegProbeResult
        .overrideOutputFiles(true) // Override the output if it exists
        .addOutput(outputName)   // Filename for the destination
        .setVideoCodec("libx265")     // Video using x264
        .setPreset("slow")
        .setConstantRateFactor(24);
final FFmpegProbeResult probe = ffprobe.probe(inputName);
```

**Expected behavior**
Expectation is that setting up the probe for the file name succeeds.

**Version (if applicable):**
 - OS: MacOS 13.4
 - Java Version 17

**Additional context**
I unfortunately am not allowed to contribute to open source projects because of my job, so I cannot create a pull request. But the fix is simple enough that it shouldn't take too much time for a contributor to fix.

Hi,
I encountered the same issue with the FFmpeg Chapter class where I had to replace the ID from `int` to `long` to make it work :

```
package net.bramp.ffmpeg.probe;

import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;

@SuppressFBWarnings(
    value = {"UUF_UNUSED_PUBLIC_OR_PROTECTED_FIELD"},
    justification = "POJO objects where the fields are populated by gson")
public class FFmpegChapter {

  public long id;
  public String time_base;
  public long start;
  public String start_time;
  public long end;
  public String end_time;
  public FFmpegChapterTag tags;
}
```
with `public int id;`, I get this Exception :
```
com.google.gson.JsonSyntaxException: java.lang.NumberFormatException: Expected an int but was 6613449456311024506 at line 206 column 38 path $.chapters[0].id
	at com.google.gson.internal.bind.TypeAdapters$7.read(TypeAdapters.java:243) ~[gson-2.9.1.jar:na]
	at com.google.gson.internal.bind.TypeAdapters$7.read(TypeAdapters.java:233) ~[gson-2.9.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:161) ~[gson-2.9.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:266) ~[gson-2.9.1.jar:na]
	at com.google.gson.internal.bind.TypeAdapterRuntimeTypeWrapper.read(TypeAdapterRuntimeTypeWrapper.java:41) ~[gson-2.9.1.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:82) ~[gson-2.9.1.jar:na]
	at com.google.gson.internal.bind.CollectionTypeAdapterFactory$Adapter.read(CollectionTypeAdapterFactory.java:61) ~[gson-2.9.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:161) ~[gson-2.9.1.jar:na]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:266) ~[gson-2.9.1.jar:na]
	at com.google.gson.Gson.fromJson(Gson.java:1058) ~[gson-2.9.1.jar:na]
	at com.google.gson.Gson.fromJson(Gson.java:986) ~[gson-2.9.1.jar:na]
	at net.bramp.ffmpeg.FFprobe.probe(FFprobe.java:116) ~[classes/:na]
	at net.bramp.ffmpeg.FFprobe.probe(FFprobe.java:47) ~[classes/:na]

```
```
Caused by: java.lang.NumberFormatException: Expected an int but was 6613449456311024506 at line 206 column 38 path $.chapters[0].id
	at com.google.gson.stream.JsonReader.nextInt(JsonReader.java:1182) ~[gson-2.9.1.jar:na]
	at com.google.gson.internal.bind.TypeAdapters$7.read(TypeAdapters.java:241) ~[gson-2.9.1.jar:na]
```

Here is the full output of ffprobe for this example :

```
C:\opt\ffmpeg\ffprobe.exe -print_format json -show_chapters "\\DiskStation\Media\Films\Revoir Pari
s (2022 1080p).mkv"
ffprobe version N-111199-g2cac3fd744-20230620 Copyright (c) 2007-2023 the FFmpeg developers
  built with gcc 13.1.0 (crosstool-NG 1.25.0.196_227d99d)
  configuration: --prefix=/ffbuild/prefix --pkg-config-flags=--static --pkg-config=pkg-config --cross-prefix=x86_64-w64-mingw32- --arch=x86_64 --target-os=mingw
32 --enable-gpl --enable-version3 --disable-debug --disable-w32threads --enable-pthreads --enable-iconv --enable-libxml2 --enable-zlib --enable-libfreetype --en
able-libfribidi --enable-gmp --enable-lzma --enable-fontconfig --enable-libvorbis --enable-opencl --disable-libpulse --enable-libvmaf --disable-libxcb --disable
-xlib --enable-amf --enable-libaom --enable-libaribb24 --enable-avisynth --enable-chromaprint --enable-libdav1d --enable-libdavs2 --disable-libfdk-aac --enable-
ffnvcodec --enable-cuda-llvm --enable-frei0r --enable-libgme --enable-libkvazaar --enable-libass --enable-libbluray --enable-libjxl --enable-libmp3lame --enable
-libopus --enable-librist --enable-libssh --enable-libtheora --enable-libvpx --enable-libwebp --enable-lv2 --enable-libvpl --enable-openal --enable-libopencore-
amrnb --enable-libopencore-amrwb --enable-libopenh264 --enable-libopenjpeg --enable-libopenmpt --enable-librav1e --enable-librubberband --enable-schannel --enab
le-sdl2 --enable-libsoxr --enable-libsrt --enable-libsvtav1 --enable-libtwolame --enable-libuavs3d --disable-libdrm --disable-vaapi --enable-libvidstab --enable
-vulkan --enable-libshaderc --enable-libplacebo --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxvid --enable-libzimg --enable-libzvbi --extra-c
flags=-DLIBTWOLAME_STATIC --extra-cxxflags= --extra-ldflags=-pthread --extra-ldexeflags= --extra-libs=-lgomp --extra-version=20230620
  libavutil      58. 13.101 / 58. 13.101
  libavcodec     60. 21.100 / 60. 21.100
  libavformat    60.  9.100 / 60.  9.100
  libavdevice    60.  2.100 / 60.  2.100
  libavfilter     9.  8.102 /  9.  8.102
  libswscale      7.  3.100 /  7.  3.100
  libswresample   4. 11.100 /  4. 11.100
  libpostproc    57.  2.100 / 57.  2.100
{
[matroska,webm @ 0000011fff6d3680] Could not find codec parameters for stream 3 (Subtitle: hdmv_pgs_subtitle (pgssub)): unspecified size
Consider increasing the value for the 'analyzeduration' (0) and 'probesize' (5000000) options
Input #0, matroska,webm, from '\\DiskStation\Media\Films\Revoir Paris (2022 1080p).mkv':
  Metadata:
    encoder         : libebml v1.4.2 + libmatroska v1.6.4
    creation_time   : 2023-01-10T11:49:23.000000Z
  Duration: 01:43:13.00, start: 0.000000, bitrate: 13744 kb/s
  Chapters:
    Chapter #0:0: start 0.000000, end 435.458000
      Metadata:
        title           : Chapitre 01
    Chapter #0:1: start 435.458000, end 852.333000
      Metadata:
        title           : Chapitre 02
    Chapter #0:2: start 852.333000, end 1437.208000
      Metadata:
        title           : Chapitre 03
    Chapter #0:3: start 1437.208000, end 1944.542000
      Metadata:
        title           : Chapitre 04
    Chapter #0:4: start 1944.542000, end 2383.042000
      Metadata:
        title           : Chapitre 05
    Chapter #0:5: start 2383.042000, end 2981.000000
      Metadata:
        title           : Chapitre 06
    Chapter #0:6: start 2981.000000, end 3473.542000
      Metadata:
        title           : Chapitre 07
    Chapter #0:7: start 3473.542000, end 3989.833000
      Metadata:
        title           : Chapitre 08
    Chapter #0:8: start 3989.833000, end 4709.833000
      Metadata:
        title           : Chapitre 09
    Chapter #0:9: start 4709.833000, end 5423.875000
      Metadata:
        title           : Chapitre 10
    Chapter #0:10: start 5423.875000, end 5936.958000
      Metadata:
        title           : Chapitre 11
    Chapter #0:11: start 5936.958000, end 6193.003000
      Metadata:
        title           : Chapitre 12
  Stream #0:0: Video: h264 (High), yuv420p(progressive), 1920x1036 [SAR 1:1 DAR 480:259], 24 fps, 24 tbr, 1k tbn (default)
    Metadata:
      BPS             : 9999579
      DURATION        : 01:43:13.000000000
      NUMBER_OF_FRAMES: 148632
      NUMBER_OF_BYTES : 7740924224
      _STATISTICS_WRITING_APP: mkvmerge v64.0.0 ('Willows') 64-bit
      _STATISTICS_WRITING_DATE_UTC: 2023-01-10 11:49:23
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
  Stream #0:1(fre): Audio: dts (DTS-HD MA), 48000 Hz, 5.1(side), s32p (24 bit) (default)
    Metadata:
      BPS             : 3730280
      DURATION        : 01:43:13.003000000
      NUMBER_OF_FRAMES: 580594
      NUMBER_OF_BYTES : 2887704508
      _STATISTICS_WRITING_APP: mkvmerge v64.0.0 ('Willows') 64-bit
      _STATISTICS_WRITING_DATE_UTC: 2023-01-10 11:49:23
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
  Stream #0:2(fre): Subtitle: subrip (default)
    Metadata:
      title           : FR FORCED
      BPS             : 80
      DURATION        : 00:00:58.500000000
      NUMBER_OF_FRAMES: 13
      NUMBER_OF_BYTES : 589
      _STATISTICS_WRITING_APP: mkvmerge v64.0.0 ('Willows') 64-bit
      _STATISTICS_WRITING_DATE_UTC: 2023-01-10 11:49:23
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
  Stream #0:3(fre): Subtitle: hdmv_pgs_subtitle (hearing impaired)
    Metadata:
      title           : FULL SDH
      BPS             : 22328
      DURATION        : 01:42:17.208000000
      NUMBER_OF_FRAMES: 2661
      NUMBER_OF_BYTES : 17129604
      _STATISTICS_WRITING_APP: mkvmerge v64.0.0 ('Willows') 64-bit
      _STATISTICS_WRITING_DATE_UTC: 2023-01-10 11:49:23
      _STATISTICS_TAGS: BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES
    "chapters": [
        {
            "id": 6613449456311024506,
            "time_base": "1/1000000000",
            "start": 0,
            "start_time": "0.000000",
            "end": 435458000000,
            "end_time": "435.458000",
            "tags": {
                "title": "Chapitre 01"
            }
        },
        {
            "id": -4433436293284298339,
            "time_base": "1/1000000000",
            "start": 435458000000,
            "start_time": "435.458000",
            "end": 852333000000,
            "end_time": "852.333000",
            "tags": {
                "title": "Chapitre 02"
            }
        },
        {
            "id": 3192285085602482051,
            "time_base": "1/1000000000",
            "start": 852333000000,
            "start_time": "852.333000",
            "end": 1437208000000,
            "end_time": "1437.208000",
            "tags": {
                "title": "Chapitre 03"
            }
        },
        {
            "id": 6990010890768538729,
            "time_base": "1/1000000000",
            "start": 1437208000000,
            "start_time": "1437.208000",
            "end": 1944542000000,
            "end_time": "1944.542000",
            "tags": {
                "title": "Chapitre 04"
            }
        },
        {
            "id": -3434126250346222931,
            "time_base": "1/1000000000",
            "start": 1944542000000,
            "start_time": "1944.542000",
            "end": 2383042000000,
            "end_time": "2383.042000",
            "tags": {
                "title": "Chapitre 05"
            }
        },
        {
            "id": 2547334430634316503,
            "time_base": "1/1000000000",
            "start": 2383042000000,
            "start_time": "2383.042000",
            "end": 2981000000000,
            "end_time": "2981.000000",
            "tags": {
                "title": "Chapitre 06"
            }
        },
        {
            "id": 8906492966509868362,
            "time_base": "1/1000000000",
            "start": 2981000000000,
            "start_time": "2981.000000",
            "end": 3473542000000,
            "end_time": "3473.542000",
            "tags": {
                "title": "Chapitre 07"
            }
        },
        {
            "id": 3300610752231630506,
            "time_base": "1/1000000000",
            "start": 3473542000000,
            "start_time": "3473.542000",
            "end": 3989833000000,
            "end_time": "3989.833000",
            "tags": {
                "title": "Chapitre 08"
            }
        },
        {
            "id": -4922550084634683681,
            "time_base": "1/1000000000",
            "start": 3989833000000,
            "start_time": "3989.833000",
            "end": 4709833000000,
            "end_time": "4709.833000",
            "tags": {
                "title": "Chapitre 09"
            }
        },
        {
            "id": 4834419182808307611,
            "time_base": "1/1000000000",
            "start": 4709833000000,
            "start_time": "4709.833000",
            "end": 5423875000000,
            "end_time": "5423.875000",
            "tags": {
                "title": "Chapitre 10"
            }
        },
        {
            "id": -8069565167791501500,
            "time_base": "1/1000000000",
            "start": 5423875000000,
            "start_time": "5423.875000",
            "end": 5936958000000,
            "end_time": "5936.958000",
            "tags": {
                "title": "Chapitre 11"
            }
        },
        {
            "id": -3248368436448126787,
            "time_base": "1/1000000000",
            "start": 5936958000000,
            "start_time": "5936.958000",
            "end": 6193003000000,
            "end_time": "6193.003000",
            "tags": {
                "title": "Chapitre 12"
            }
        }
    ]
}
```

This test was made on `Windows 10` / `java 17.0.7` / `ffprobe version N-111199-g2cac3fd744-20230620`

First chapter item is :
```
    "chapters": [
        {
            "id": 6613449456311024506,
            "time_base": "1/1000000000",
            "start": 0,
            "start_time": "0.000000",
            "end": 435458000000,
            "end_time": "435.458000",
            "tags": {
                "title": "Chapitre 01"
            }
        },
```

with `"id": 6613449456311024506`

Regards.

--- a/src/main/java/net/bramp/ffmpeg/probe/FFmpegChapter.java
+++ b/src/main/java/net/bramp/ffmpeg/probe/FFmpegChapter.java
@@ -7,7 +7,7 @@ import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
     justification = "POJO objects where the fields are populated by gson")
 public class FFmpegChapter {
 
-  public int id;
+  public long id;
   public String time_base;
   public long start;
   public String start_time;

--- a/src/test/java/net/bramp/ffmpeg/FFprobeTest.java
+++ b/src/test/java/net/bramp/ffmpeg/FFprobeTest.java
@@ -53,6 +53,9 @@ public class FFprobeTest {
     when(runFunc.run(argThatHasItem(Samples.side_data_list)))
         .thenAnswer(new NewProcessAnswer("ffprobe-side_data_list"));
 
+    when(runFunc.run(argThatHasItem(Samples.chapters_with_long_id)))
+        .thenAnswer(new NewProcessAnswer("chapters_with_long_id.m4b"));
+
     ffprobe = new FFprobe(runFunc);
   }
 
@@ -159,5 +162,13 @@ public class FFprobeTest {
         is(
             "\n00000000:            0      -65536           0\n00000001:        65536           0           0\n00000002:            0           0  1073741824\n"));
     assertThat(info.getStreams().get(0).side_data_list[0].rotation, is(90));
+  }
+
+  @Test
+  public void testChaptersWithLongIds() throws IOException {
+    FFmpegProbeResult info = ffprobe.probe(Samples.chapters_with_long_id);
+
+    assertThat(info.getChapters().get(0).id, is(6613449456311024506L));
+    assertThat(info.getChapters().get(1).id, is(-4433436293284298339L));
   }
 }

--- a/src/test/java/net/bramp/ffmpeg/fixtures/Samples.java
+++ b/src/test/java/net/bramp/ffmpeg/fixtures/Samples.java
@@ -31,6 +31,8 @@ public final class Samples {
 
   public static final String divide_by_zero = FAKE_PREFIX + "Divide By Zero.mp4";
 
+  public static final String chapters_with_long_id = FAKE_PREFIX + "chapters_with_long_id.m4b";
+
   // TODO Change to a temp directory
   // TODO Generate random names, so we can run tests concurrently
   public static final String output_mp4 = "output.mp4";

--- a/src/test/resources/net/bramp/ffmpeg/fixtures/chapters_with_long_id.m4b
+++ b/src/test/resources/net/bramp/ffmpeg/fixtures/chapters_with_long_id.m4b
@@ -1,0 +1,134 @@
+{
+    "streams": [
+        {
+            "index": 0,
+            "codec_name": "aac",
+            "codec_long_name": "AAC (Advanced Audio Coding)",
+            "profile": "LC",
+            "codec_type": "audio",
+            "codec_time_base": "1/44100",
+            "codec_tag_string": "mp4a",
+            "codec_tag": "0x6134706d",
+            "sample_fmt": "fltp",
+            "sample_rate": "44100",
+            "channels": 1,
+            "channel_layout": "mono",
+            "bits_per_sample": 0,
+            "r_frame_rate": "0/0",
+            "avg_frame_rate": "0/0",
+            "time_base": "1/44100",
+            "start_pts": 0,
+            "start_time": "0.000000",
+            "duration_ts": 248628224,
+            "duration": "5637.828209",
+            "bit_rate": "62993",
+            "max_bit_rate": "75648",
+            "nb_frames": "242801",
+            "disposition": {
+                "default": 1,
+                "dub": 0,
+                "original": 0,
+                "comment": 0,
+                "lyrics": 0,
+                "karaoke": 0,
+                "forced": 0,
+                "hearing_impaired": 0,
+                "visual_impaired": 0,
+                "clean_effects": 0,
+                "attached_pic": 0,
+                "timed_thumbnails": 0
+            },
+            "tags": {
+                "creation_time": "2016-02-21T08:27:56.000000Z",
+                "language": "und"
+            }
+        },
+        {
+            "index": 1,
+            "codec_name": "bin_data",
+            "codec_long_name": "binary data",
+            "codec_type": "data",
+            "codec_tag_string": "text",
+            "codec_tag": "0x74786574",
+            "r_frame_rate": "0/0",
+            "avg_frame_rate": "0/0",
+            "time_base": "1/44100",
+            "start_pts": 0,
+            "start_time": "0.000000",
+            "duration_ts": 248628224,
+            "duration": "5637.828209",
+            "bit_rate": "1",
+            "nb_frames": "24",
+            "disposition": {
+                "default": 0,
+                "dub": 0,
+                "original": 0,
+                "comment": 0,
+                "lyrics": 0,
+                "karaoke": 0,
+                "forced": 0,
+                "hearing_impaired": 0,
+                "visual_impaired": 0,
+                "clean_effects": 0,
+                "attached_pic": 0,
+                "timed_thumbnails": 0
+            },
+            "tags": {
+                "creation_time": "2016-02-21T08:28:13.000000Z",
+                "language": "und"
+            }
+        }
+    ],
+    "chapters": [
+        {
+            "id": 6613449456311024506,
+            "time_base": "1/1000000000",
+            "start": 0,
+            "start_time": "0.000000",
+            "end": 435458000000,
+            "end_time": "435.458000",
+            "tags": {
+                "title": "Chapitre 01"
+            }
+        },
+        {
+            "id": -4433436293284298339,
+            "time_base": "1/1000000000",
+            "start": 435458000000,
+            "start_time": "435.458000",
+            "end": 852333000000,
+            "end_time": "852.333000",
+            "tags": {
+                "title": "Chapitre 02"
+            }
+        }
+    ],
+    "format": {
+        "filename": "sammyjay_1602_librivox.m4b",
+        "nb_streams": 2,
+        "nb_programs": 0,
+        "format_name": "mov,mp4,m4a,3gp,3g2,mj2",
+        "format_long_name": "QuickTime / MOV",
+        "start_time": "0.000000",
+        "duration": "5637.828209",
+        "size": "45417172",
+        "bit_rate": "64446",
+        "probe_score": 100,
+        "tags": {
+            "major_brand": "M4A",
+            "minor_version": "0",
+            "compatible_brands": "3gp5isom",
+            "creation_time": "2016-02-21T08:27:56.000000Z",
+            "genre": "Audiobook",
+            "media_type": "2",
+            "encoder": "Chapter and Verse V 1.5",
+            "title": "The Adventures of Sammy Jay",
+            "artist": "Thornton W. Burgess",
+            "album": "The Adventures of Sammy Jay",
+            "comment": "https://archive.org/details/sammyjay_1602_librivox",
+            "copyright": "PD1.0",
+            "track": "1",
+            "Encoding Params": "vers"
+        }
+    }
+}

